--- src/app/core/constants/attestations-per-format.ts	2026-03-06 18:15:57
+++ src/app/core/constants/attestations-per-format.local.ts	2026-03-06 18:16:51
@@ -1,5 +1,5 @@
 import {Attestation, MsoMdocAttestation, SdJwtVcAttestation} from "@core/models/attestation/Attestations";
-import {EHIC_ATTESTATION, EHIC_ATTESTATION_DC4EU, MDL_ATTESTATION, PDA1_ATTESTATION, PHOTO_ID_ATTESTATION, PID_ATTESTATION, LEARNING_CREDENTIAL_ATTESTATION} from "@core/constants/attestation-definitions";
+import {EHIC_ATTESTATION, EHIC_ATTESTATION_DC4EU, MDL_ATTESTATION, PDA1_ATTESTATION, PHOTO_ID_ATTESTATION, PID_ATTESTATION, LEARNING_CREDENTIAL_ATTESTATION, ACADEMIC_CREDIT_ATTESTATION} from "@core/constants/attestation-definitions";
 import {AttestationFormat} from "@core/models/attestation/AttestationFormat";
 import {AttestationType} from "@core/models/attestation/AttestationType";
 import {DataElement} from "@core/models/attestation/AttestationDefinition";
@@ -87,6 +87,14 @@
   claimQuery: (attribute: DataElement) => { return { path: sdJwtVcAttributeClaimQuery(attribute, AttestationType.LEARNING_CREDENTIAL) } }
 }
 
+/*---- ACADEMIC CREDIT INSTANCES PER FORMAT ----*/
+export const ACADEMIC_CREDIT_SD_JWT_VC: SdJwtVcAttestation = {
+  format: AttestationFormat.SD_JWT_VC,
+  attestationDef: ACADEMIC_CREDIT_ATTESTATION,
+  vct: "urn:com:dentsusoken:academic_credit:1",
+  claimQuery: (attribute: DataElement) => { return { path: [attribute.identifier] } }
+}
+
 function resolveAttribute(attribute: DataElement, attestationType: AttestationType): string {
   let resolvedAttribute = attribute.identifier
   if (attestationType === AttestationType.PID) {
@@ -141,7 +149,7 @@
 
 export const ATTESTATIONS_BY_FORMAT: { [id: string]: Attestation[] } = {
   "mso_mdoc": [PID_MSO_MDOC, MDL_MSO_MDOC, PHOTO_ID_MSO_MDOC, EHIC_MSO_MDOC, PDA1_MSO_MDOC],
-  "dc+sd-jwt": [PID_SD_JWT_VC, EHIC_SD_JWT_VC, PDA1_SD_JWT_VC, EHIC_SD_JWT_VC_DC4EU, LEARNING_CREDENTIAL_SD_JWT_VC]
+  "dc+sd-jwt": [PID_SD_JWT_VC, EHIC_SD_JWT_VC, PDA1_SD_JWT_VC, EHIC_SD_JWT_VC_DC4EU, LEARNING_CREDENTIAL_SD_JWT_VC, ACADEMIC_CREDIT_SD_JWT_VC]
 }
 
 export const getAttestationByFormatAndType =
