--- src/app/core/models/attestation/AttestationType.ts	2026-03-06 18:15:36
+++ src/app/core/models/attestation/AttestationType_local.ts	2026-03-11 17:04:19
@@ -5,5 +5,7 @@
   EHIC = "ehic",
   EHIC_DC4EU = "ehic-dc4eu",
   PDA1 = "pda1",
-  LEARNING_CREDENTIAL = "learning_credential"
+  LEARNING_CREDENTIAL = "learning_credential",
+  ACADEMIC_CREDIT = "academic_credit",
+  UNIVERSITY_STUDENT = "university_student"
 }
